app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -138,23 +187,62 @@ def process_availability(availability)
       end
 
       def parse_deployment(server, deployment)
-        {
+        specific = {
           :name              => parse_deployment_name(deployment.id),
-          :middleware_server => server, # TODO: does that make sense? What is better?
-          :nativeid          => deployment.id,
-          :ems_ref           => deployment.path
+          :middleware_server => server,
         }
+        parse_base_item(deployment).merge(specific)
```

Comment:
any reason for not using `merge! `?

also, I find "specific" not a very clear name. perhaps "additional_properties" would be more clear, I am open to hear other suggestions.